src/styles/base/footer.scss

```diff
@@ -1,12 +1,22 @@
 .footer {
     padding: 40px 0;
+    font-size: 14px;
+    line-height: urset;
```

Comment:
I think there is no value as `urset`. Did you mean `unset`?

Reply:
Done!